### ISSUE TITLE
Replicating paper evaluation results

### ISSUE THREAD

**Issue by hturki** [princeton-computational-imaging/neural-scene-graphs#14]
Thanks for the paper and making your code public. I had some questions around reproducing the results in Table 1 of the paper:

- To confirm, is the training sequence is some set of input frames from a KITTI sequence (frames 65 to 120 for Scene 06) and then are the evaluation numbers reported by rendering all of the training set frames (ie: is the validation set identical to the training set?) or some subset
- Is the evaluation script present in this repo? Running ```python main.py --config example_configs/config_kitti_0006_example_render.txt``` seems to render 3 frames + a video of those 3 frames, but I'm not sure that it's showing any evaluation metrics. 
- Are the numbers reported against subsequences of scenes 0001, 0002, and 0006? If so, would it be possible to learn the starting and ending frames used for 0001 and 0002? The README mentions config files for those scenes but I don't see them in the repo
